### PR TITLE
#10539 layout.rst

### PR DESCRIPTION
**Pull request by Konntroll**
Finished editing layout.rst for task #10539.

Please let me know if you would like me to make any changes to the text.

### CODE REVIEW

File: sphinx/source/docs/user_guide/layout.rst [hunk 1]
```diff
@@ -1,73 +1,74 @@
...
+Grid layout for plots
 ~~~~~~~~~~~~~~~~~~~~~~
 
-The |gridplot| function can be used to arrange
-Bokeh Plots in a grid layout. |gridplot| also collects all
-tools into a single toolbar, and the currently active tool is the same
-for all plots in the grid. It is possible to leave "empty" spaces in
-the grid by passing ``None`` instead of a plot object.
+Use the |gridplot| function to arrange Bokeh plots in a grid. This 
+funciton also merges all plot tools into a single toolbar. Each plot
```
Comment (tcmetzger) [inline sphinx/source/docs/user_guide/layout.rst:40]: Should be "function", not "funciton"

File: sphinx/source/docs/user_guide/layout.rst [hunk 2]
```diff
@@ -1,73 +1,74 @@
...
+number of columns you want in your grid. For example:
 
 .. code-block:: python
 
     gridplot([s1, s2, s3], ncols=2)
 
-In addition, you can pass in ``plot_width`` and ``plot_height`` arguments,
-and these will be used to set the size of all your plots.
+You can also pass in ``plot_width`` and ``plot_height`` arguments.
+These dimensions will then appply to all your plots.
```
Comment (tcmetzger) [inline sphinx/source/docs/user_guide/layout.rst:57]: Should be "apply", not "appply"

File: sphinx/source/docs/user_guide/layout.rst [hunk 3]
```diff
@@ -80,122 +81,115 @@ for quickly creating layouts:
...
 ``"fixed"``
-    Component is not responsive. It will retain its original width and height
-    regardless of any subsequent browser window resize events.
+    Component retains its width and height regardless of browser window size.
 
 ``"stretch_width"``
-    Component will responsively resize to stretch to the available width, without
-    maintaining any aspect ratio. The height of the component depends on the type
-    of the component and may be fixed or fit to component's contents.
+    Component resizes to fill availble width but does not maintain any aspect
```
Comment (tcmetzger) [inline sphinx/source/docs/user_guide/layout.rst:108]: Should be "available", not "availble "

File: sphinx/source/docs/user_guide/layout.rst [hunk 4]
```diff
@@ -80,122 +81,115 @@ for quickly creating layouts:
...
+    Component resizes to fill availble width but does not maintain any aspect 
+    ratio. Height depends on the component type and may fit its contents or be
+    fixed.
 
 ``"stretch_height"``
-    Component will responsively resize to stretch to the available height, without
-    maintaining any aspect ratio. The width of the component depends on the type
-    of the component and may be fixed or fit to component's contents.
-
+    Component resizes to fill availble height but does not maintain any aspect
```
Comment (tcmetzger) [inline sphinx/source/docs/user_guide/layout.rst:113]: Should be "available", not "availble "

File: sphinx/source/docs/user_guide/layout.rst [hunk 5]
```diff
@@ -80,122 +81,115 @@ for quickly creating layouts:
...
+    
+Depending on the mode, you may also have to specify ``width`` and/or ``height``.
+For example, you have to specify a fixed height when using the ``stretch_width`` 
+mode.
 
-Note that layout objects such as rows and columns will pass on their configured
-sizing mode to any of their children that do not themselves have an explicitly
-set ``sizing_mode`` of their own.
+Components such as |row| and |column| elements share their sizing mode with all
+of their children that do not have their own excplicit sizing mode.
```
Comment (tcmetzger) [inline sphinx/source/docs/user_guide/layout.rst:138]: Should be "explicit", not "excplicit"

**Comment by tcmetzger** [comment]
@Konntroll thanks again for some major improvements to the text. I just saw a few typos to fix. 
It would be helpful if you could remove the file jupyter.rst from this pull request. This pr should only contain layout.rst.

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @Konntroll !

**Pull request merged**